.github/workflows/ci_build_android.yml

```
@@ -0,0 +1,50 @@
+name: Build Android library
+
+on:
+  workflow_call:
+    outputs:
+      artifact-dir-base:
+        description: The base name of artifact directory (without the target suffix)
+        value: ${{ jobs.build-android-libs.outputs.artifact-dir-base }}
+
+jobs:
+  build-android-libs:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 35
+    strategy:
+      matrix:
+        target: ["arm64-v8a", "x86_64"]
```

In cargo ndk we can also use the [rust triple](https://github.com/bbqsrc/cargo-ndk/blob/f91b56d4114dd18fa7d2fa3b484fb214288e6450/src/meta.rs#L88). 
```suggestion
        target: ["aarch64-linux-android", "x86_64-linux-android"]
```

---

On the CI we want build for all the android targets: aarch64-linux-android, armv7-linux-androideabi, i686-linux-android, x86_64-linux-android

---

Mmh, I think I was told that we don't support armv7. I'm assuming that information is incorrect, then? Also what's the point of building Intel non-64-bit architectures?

---

for ios we don't support armv7 because 32Bit is no longer supported since ios 11 i think

---

Someone still have an emulator that wants the non 64 version.